bouncer/authentication/authentication.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import User
+from rest_framework import authentication
+from rest_framework import exceptions
+from authentication import models
+
+
+class Authentication(authentication.TokenAuthentication):
```

Comment:
Please rename this to `TokenAuthentication`, to be more indicative of how it works.